backend/requirements.txt

```diff
@@ -34,4 +34,4 @@ django-jet==1.0.7
 #######################################################
 ### Production                                      ###
 #######################################################
-uWSGI==2.0.13.1
+uWSGI==2.0.18
```

Comment:
This really necessary? Maybe in a separate commit?

Reply:
LGTM - `2.0.13.` is from 20160513... Lots of bugs have been fixed since.